Makefile

```diff
@@ -2,6 +2,29 @@ E2E_FOLDER ?= e2e
 OPERATOR_FOLDER ?= operator
 SERVER_FOLDER ?= server
 
+MDBOOK_VERSION ?= v0.4.40
+BOOK_PATH = $(PWD)/doc/book
+
+.PHONY: doc
+doc:
+	docker run -it --rm \
```

Comment:
Could we have this run through a configurable container runtime instead of hardcoding this to docker?  I largely have migrated to `podman` nowadays.

Reply:
sure, good idea. I'll parametrize this

Reply:
done in 189aeb3a8c9d2b6fc54c980f52acb13d51d77580